Include/internal/pycore_object.h

```diff
@@ -31,6 +31,16 @@ PyAPI_FUNC(void) _Py_NO_RETURN _Py_FatalRefcountErrorFunc(
 
 #define _Py_FatalRefcountError(message) _Py_FatalRefcountErrorFunc(__func__, message)
 
+/// Increment reference counter by specified number of items
+static inline void Py_INCREF_n(PyObject* op, Py_ssize_t n)
+{
+#ifdef Py_REF_DEBUG
+    _Py_RefTotal+=n;
+#endif
+    op->ob_refcnt+=n;
+}
+#define Py_INCREF_n(op, n) Py_INCREF_n(_PyObject_CAST(op), n)
```

Comment:
This feature is used by some 3rd party projects (not sure if they update _Py_RefTotal correctly). It would be interesting to add it to the public C API. But I dislike the function, "INC" in INCREF means "increment": "add one". I suggest renaming the function to ``Py_RefcntAdd(obj, n)``.

Reply:
Since this is internal API, it should be prefixed with `_`.